lib/TelemetryFeed.jsm

```diff
@@ -844,7 +844,7 @@ this.TelemetryFeed = class TelemetryFeed {
    */
   handleDiscoveryStreamSpocsFill(data) {
     const payload = this.createSpocsFillPing(data);
-    this.sendStructuredIngestionEvent(payload, "spocs-fills", "1");
+    this.sendStructuredIngestionEvent(payload, "spoc-fills", "1");
```

Comment:
Should we change name here as well https://github.com/mozilla/activity-stream/blob/master/common/Actions.jsm#L299

Reply:
Aha, good catch. Yes, let me clean it up a bit. Naming stuff is really hard ;)